python/ray/util/dask/common.py

```diff
@@ -7,7 +7,12 @@
 
 from dask.base import quote
 from dask.core import get as get_sync
-from dask.utils import apply
+
+try:
```

Comment:
We don't need this anymore. `from dask.utils import apply` should already be backwards compatible.

Reply:
wait so we don't need this PR at all?

Reply:
Yes if we want to add to the docs. But no code change is needed.